server.js

```diff
@@ -23,3 +23,38 @@ app.get('/api/v1/all-products', async (request, response) => {
   }
 });
 
+
+// app.post('/api/v1/favorite-product', async (request, response) => {
+//   const product = request.body;
+
+//   if(!product.id) {
+//     response
+//       .status(422)
+//       .send({ error: 'Id is required.'});
+//   }
+
+//   try {
+//     const id = await database('product').insert(product, 'id');
+//     response.status(201).json({ id })
+//   } catch (error) {
+//     response.status(500).json({ error });
+//   }
+// });
+
+app.put('/api/v1/favorite/:id', async (request, response) => {
+  try {
+    const { id } = request.params;
+    const product = await database('product')
+      .where('id', '=', id)
```

Comment:
What do you think about moving line 48 back to 47 or can you please clarify why it needs to be on it's own line?

Reply:
I put 48 on it's own line because it's chaining the method above it and I find it more readable, if you would prefer to have it on the same line it can be moved!